OpenRA.Mods.Common/Activities/Move/MoveAdjacentTo.cs

```diff
@@ -161,6 +161,11 @@ public override IEnumerable<Target> GetTargets(Actor self)
 			return Target.None;
 		}
 
+		public override IEnumerable<TargetLineNode> TargetLineNodes(Actor self)
+		{
+			yield break;
```

Comment:
Can you please pass the desired colour in via the constructor so that you can render the line here?
You can then remove the ugly workarounds that are needed to fix the line rendering on the next activity in the queue.

Reply:
I doubt this is a walkaround once the activities are sorted out. MoveAdjacentTo is likely to be a child activity that isn't supposed to be traversed by DrawLinetoTarget.cs, I think.

https://github.com/OpenRA/OpenRA/pull/14384/files/c37d0abc59d875fc206edd092ce53f5c686c3bb0#diff-d7a57eebc0a2b3d470e099590d38922eR69

Then line 69 of DrawLineToTarget.cs can safely become for (; a != null; a = a.NextInQueue).

Reply:
Likely but not guaranteed.  Making `MoveAdjacentTo` differently to most other activities and then adding a workaround for that in `CallFunc` is almost certainly going to turn into a maintenance problem looking forward.